packages/mobile/src/verify/sagas/resetKomenciSessionSaga.ts

```diff
@@ -0,0 +1,10 @@
+import { navigate } from 'src/navigator/NavigationService'
+import { Screens } from 'src/navigator/Screens'
+
+// Should this not be part of the saga? When we find out a user ran out of Komenci quota we need
+// them to redo the reCaptcha. To do that they need to go back to the VerificationEducationScreen.
+// When this happens they are usually on the VerificationInputScreen, so this brings them back so
+// they have to press on 'start' again and they get the reCaptcha prompt.
+export default function resetKomenciSessionSaga() {
+  navigate(Screens.VerificationEducationScreen)
+}
```

Comment:
The main `saga` file has 900+ lines, so I didn't want to put more stuff there. I would like to move all sagas each to their different file. Ivan had started doing this but only did it for a few, it'd be ideal to have each saga on its own file which would tidy up the folder structure imo.

Since I haven't moved the others yet I could move this to the main saga file for now if you prefer